src/WebJobs.Extensions.Http/Routing/IWebJobsRouter.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.AspNetCore.Routing;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Http
+{
+    public interface IWebJobsRouter : IRouter
```

Comment:
I applied the ***Http*** changes proposed, but it feels a bit redundant. We can revisit if needed.